### DISCUSSION TITLE
Error in PhysioNet2012 example

### DISCUSSION THREAD

**Discussion by jlara6** [WenjieDu/PyPOTS#36] [General]
I tried the SAITS application repository example in PyPOTS to impute missing values in the PhysioNet2012 dataset, however. I got the error: 

Loading the dataset physionet_2012 with TSDB (https://github.com/WenjieDu/Time_Series_Database)...
Starting preprocessing physionet_2012...
Start downloading...
Exception: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)>
Download failed. Aborting.

How could I solve it ?  Since I would like to use the library with other data, but I want to understand the example of the repository.

**Comment by WenjieDu** [comment]
Hi Jorge, I think this is caused by a networking issue. You can try again by using a network proxy.

And thank you so much for your attention to PyPOTS! If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

**Reply by jlara6** [reply]
I'm going to try it, don't you have another database to run the library, because I checked the code and it is only for PhysioNet2012

**Reply by WenjieDu** [reply]
This example is only to help you get to know how PyPOTS works. You can use other datasets once you know how to apply it.
